package.json

```diff
@@ -61,7 +61,10 @@
     "@justeat/pie-design-tokens": "4.2.0",
     "@justeat/stylelint-config-fozzie": "3.x",
     "@percy/cli": "1.20.1",
+    "@percy/playwright": "1.0.4",
     "@percy/webdriverio": "2.0.2",
+    "@playwright/test": "1.31.2",
+    "@sand4rt/experimental-ct-web": "1.0.7",
```

Comment:
is this a playwright depedency?

Reply:
Yes, this is a 3rd party playwright component testing dependency that we're using till Playwright adds in formal component testing which is currently being worked on.